packages/x-handlebars/concerns/resolve-peer.js

```diff
@@ -0,0 +1,2 @@
+const path = require('path');
+module.exports = (moduleId) => path.join(process.cwd(), 'node_modules', moduleId);
```

Comment:
should this be in a common dependency of x-handlebars and x-engine? or exported from x-engine?

Reply:
If we need it again then I think making it a part of x-engine would be sensible.